File: film-information-service-interface/src/main/java/com/michael/service/EvaluateService.java
package com.michael.service;

import com.michael.pojo.Evaluate;
import com.michael.pojo.User;
import entity.PageResult;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface EvaluateService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<Evaluate> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(Evaluate evaluate);

	

	/**
	 * 根据ID获取实体
	 * @param evaluate_id
	 * @return
	 */
	public Evaluate findOne(Long evaluate_id);


	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(Evaluate evaluate, int pageNum, int pageSize);


	/**
	 *
	 * @return
	 */
	public Long countNumber();
	
}

File: film-information-backstage-web/src/main/webapp/js/service/evaluateService.js
//服务层
app.service('evaluateService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../evaluate/findAll.do');
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../evaluate/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../evaluate/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../evaluate/add.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../evaluate/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../evaluate/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});
